pkg/k8s/test/integration/cluster_summary_test.go

```diff
@@ -0,0 +1,60 @@
+package integration
```

Comment:
```suggestion
//go:build k8s_integration
package integration
```

Can you please add a build tag as below so that this test will not be run in `mage test:integration`?
https://github.com/aquasecurity/trivy/blob/48e021ea6bf21125d92cb3562b74db121972e17a/integration/module_test.go#LL1C1-L1C1